### PR TITLE
youtube stuff

### PR DESCRIPTION
**Pull request by shankrohan**

### CODE REVIEW

File: backend/services/youtubeService.js [hunk 1]
```diff
@@ -97,6 +98,67 @@ exports.likedVideos = async function (req, res) {
...
+      maxResults: 10,
+      myRating: 'like', 
+    });
+
+    //console.log('result: ' + result)
+    var vidList = result.data.items
+    const catArr = [] 
+      var i = 0
+      vidList.forEach(vid => {
+        catArr[i] = vid.snippet.categoryId
```
Comment (tappenze) [inline backend/services/youtubeService.js:116]: would be better to just use .push()

File: backend/services/youtubeService.js [hunk 2]
```diff
@@ -97,6 +98,67 @@ exports.likedVideos = async function (req, res) {
...
+        console.log('CATEGORY ID: ' + vid.snippet.categoryId)
+        i++
+      });
+
+      const maxCat = mode(catArr)
```
Comment (tappenze) [inline backend/services/youtubeService.js:121]: would also be a little more functional to use a hashmap here but for the sake of time its not necessary

**Review by tappenze — COMMENTED** [review]
in service, what is subcount/cid and what is mostSubscribers?

**Comment by shankrohan** [comment]
> in service, what is subcount/cid and what is mostSubscribers?

mostSubscribers is intended to return the top 5 channels based on subscriptionCount (number of subscribers the channel has). The choice is made from all of the channels that the current user is subscribed to.

I used subCount as a function to return the subscription counts but since it is happening in a loop it's not working correctly. 

cid is all the unique channelIds that the current user is subscribed to.

File: backend/services/youtubeService.js (hunk 1 above)
Comment (shankrohan) [reply backend/services/youtubeService.js:116]: i can fix that in my next push.

**Review by tappenze — APPROVED** [review]

**Pull request merged**